### DISCUSSION TITLE
Demo site

### DISCUSSION THREAD

**Discussion by fredericocarneiro** [AppFlowy-IO/AppFlowy#1366] [General]
Hi there!
Is there any demo site where I can check it out before worrying about installing it?
I saw some documentation in gitbook but why Appflowy doesn't use itself as a documenting tool?

Thanks

**Comment by annieappflowy** [comment]
Hi @fredericocarneiro , thanks for asking. We don't support Web yet. At the moment, we only offer downloadable packages for Linux, macOS, and Windows. I'm considering a demo site next quarter for people to try out without installing.
